portfolio/src/main/webapp/style.css

```diff
@@ -83,26 +87,42 @@ body{
   margin-bottom:10px
 }
 
+
 .cardSecondary{
 }
 
 .cardLongDesc{
   display: none;
+  opacity: 0.0;
 }
 
 .openCard{
-  width:70vw;
-  min-height: 300px;
+  /*min-height: 300px;*/
 }
 
 .openCard .cardImg{
-  display: none;
+  height: 0px;
+  /*animation: openCardImg 0.2s linear 0s forwards;*/
 } 
 
 .openCard .cardLongDesc{
   display: block;
+  animation: openCardLongDesc 0.2s ease-in 0.5s forwards;
 } 
 
+@keyframes openCardImg{
+  to{height: 0px;}
```

Comment:
I would explicitly specify the `from` keyframe as well. I don't even know if this will work repeatedly without it.